piece_of_cake.py

```diff
@@ -0,0 +1,17 @@
+def get_recipe_price(prices=None, optionals=None, **ingredients):
+
+    if prices is not None and ingredients is not None and len(ingredients) == len(prices):
```

Comment:
You dont have to check if prices is not None - just assume the user is not stupid - check if its not empty though and then return a matching message - or raise a matching exception